### DISCUSSION TITLE
WLAN Settings

### DISCUSSION THREAD

**Discussion by colestar99** [FanatiQS/wireless_atem_camera_control#1] [Q&A]
Unfortunately it does not work so far. Where or in which file can I set my Wifi settings.
I haven't found it yet.

**Comment by FanatiQS** [comment]
Terribly sorry I'm so late to answering. Didn't get a notification for some reason.
Are you still having issues getting it to work?

**Comment by colestar99** [comment]
Hello Gustav,
I have successfully implemented this project so far. it also works well. Great work.
But I have the following questions/suggestions:

- Is the atem time code also transmitted?
- Is the REF also transferred?
- Would it be possible to make the tally output switchable via software / HTML web interface - status? I.e. select which status you want to output - PGM/PVW, PGM only, OFF. Thank you....

regards Stefan

**Reply by FanatiQS** [reply]
Hi Stefan,

Great to hear and thank you.

What do you mean by breathing time?

As far as I know, ATEM does not transfer the the REF signal over the network. It would require a dedicated transmitter that receives the REF signal and sends it over the network and might still not be as accurate as using a cable depending on network setup. I would suggest jam-syncing the camera at the beginning and maybe again a few hours later. Though I don't know how well blackmagics cameras internal clocks are, I would assume they don't drift more than a few milliseconds an hour.

Yes, that would be possible to add. I remember considering that a while back but never implemented it.

If you haven't checked out the license section, keep in mind that to use this project in production you have to reach out for permission. It is only free to test out and evaluate. You can reach out with a private email, this thread is going through a github discussion.
